azext_iot/tests/test_iot_ext_int.py

```diff
@@ -506,6 +506,15 @@ def test_hub_modules(self):
                          self.exists('authentication.symmetricKey.primaryKey'),
                          self.exists('authentication.symmetricKey.secondaryKey')])
 
+        # Error can't get a sas token for module without device
+        self.cmd('az iot hub generate-sas-token -n {} -g {} -m {}'
+                 .format(LIVE_HUB, LIVE_RG, module_ids[1]), expect_failure=True)
+
+        # sas token for module
+        self.cmd('iot hub generate-sas-token -n {} -g {} -d {} -m {}'
```

Comment:
Remember to add a test to generate-sas-token with connection string like [here](https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/tests/test_iot_ext_int.py#L358).

Reply:
Added